### PR TITLE
resurrect the shariff.complete.js

### PR DESCRIPTION
**Pull request by liayn**
Followup to #214 to re-establish the `shariff.complete.js` file.
The js files now contain the content as expected:
- `shariff.complete.js` includes the custom `dom` module
- `shariff.min.js` relies on jQuery to be present

### CODE REVIEW

**Comment by liayn** [comment]
@compeak @glaszig please take a look

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Changelog
 
+## 1.25.2 2017-
+
+* Restored the jQuery dependency for the min.js version (liayn)
```
Comment (compeak) [inline CHANGELOG.md]: Please change this to

`Restored the jQuery dependency for the min.js version. (liayn)`

Comment (liayn) [inline CHANGELOG.md]: done

**Comment by compeak** [comment]
Just to make sure I don't screw up again: This is ready to merge, right?

**Comment by liayn** [comment]
@compeak give me some time please to review everything.

**Comment by liayn** [comment]
I rebased the patch on latest master. It stills needs a bit more of testing. Maybe @glaszig can help out with testing too?

**Comment by glaszig** [comment]
a quick idea/suggestion first. maybe the distributed files should follow the naming convention of

- `shariff.jquery.js` for use with jquery -- meaning jquery being required to run (like with jquery plugins)
- `shariff.js` bundled with the shariff-native dom module

plus a `min` version for both of them?

my reasoning is that `.jquery.js` is commonly known from jquery plugins requiring jquery to be included first.

**Comment by liayn** [comment]
The problem is that we want to avoid a breaking change. Renaming files would be such a breaking change. In general I like the idea, but maybe lets do that after a v1.25.x release as v2.0.0

**Comment by liayn** [comment]
done and rebased.

@compeak Why is the build folder then not excluded via gitignore??

**Comment by compeak** [comment]
> Why is the build folder then not excluded via gitignore??

 Because we keep it in version control as a convenience for people who download the repository instead of releases. And because I cannot read minified code, I have to treat it as untrusted. This means that build artifacts that have not been build by me or other team members may not be committed into the repository. (Yes, we handled that differently in the beginning.)

**Comment by liayn** [comment]
@compeak

> build artifacts that have not been build by me or other team members may not be committed

Lets add that to some contribution guidelines ;-)

**Comment by compeak** [comment]
Thanks.

> Lets add that to some contribution guidelines ;-)

I tried to spare myself from writing guidelines with the pull request template. But maybe that's too subtle.

**Pull request merged**